skbeam/core/correlation.py

```diff
@@ -915,29 +915,43 @@ def _validate_and_transform_inputs(num_bufs, num_levels, labels):
     )
 
 
-def one_time_from_two_time(two_time_corr):
+def one_time_from_two_time(two_time_corr, calc_errors=False):
     """
     This will provide the one-time correlation data from two-time
-    correlation data.
+    correlation data. An estimator for errors can be calculated
+    according to the central limit theorem.
 
     Parameters
     ----------
     two_time_corr : array
         matrix of two time correlation
         shape (number of labels(ROI's), number of frames, number of frames)
+    calc_errors: boolean
+        True to calculate error bars, False to not calculate error bars
 
     Returns
     -------
     one_time_corr : array
         matrix of one time correlation
         shape (number of labels(ROI's), number of frames)
+    err_one_time_corr: array
+        matrix of errors for one time correlation
+        shape (number of labels(ROI's), number of frames)
```

Comment:
This part is conditional, it may or may not be returned. Should it be mentioned?

Reply:
added this

Reply:
Also added the error bar test for the previous version test.